internal/config/releases.go

```diff
@@ -107,3 +108,14 @@ func GetLatestReleases(releases uint) ([]*Release, error) {
 
 	return releaseList, nil
 }
+
+// GetFileNames takes the url of a github repo and the path to a directory. Then returns all the filenames from that directory
+func GetFileNames(owner string, repo string, path string) ([]string, error) {
+	g := walker.NewGit()
+	var filenames []string
+	err := g.Owner(owner).Repo(repo).Branch("master").Root(path).RegisterFileInterceptor(func(f walker.File) error {
+		filenames = append(filenames, f.Name)
```

Comment:
Assuming that this is concurrent in nature, won't this cause race issue?

Reply:
how does the concurrent nature can cause filenames array to be not what we expect by race condition?
Can you give an example?

Reply:
Slice after two appends => `[filename1][filename2][][]`
3 (or more) go routines try to append at the same time, all of them query for the capacity - all of them think they can add the value. Who wins?

Reply:
This is fixed, yeah?